Rubberduck.VBEEditor/Events/KeyPressEventArgs.cs

```diff
@@ -15,8 +15,19 @@ public KeyPressEventArgs(IntPtr hwnd, IntPtr wParam, IntPtr lParam, bool keydown
 
             if (keydown)
             {
-                // Why \r and not \n? Because it really doesn't matter...
-                Character = ((Keys)wParam & Keys.KeyCode) == Keys.Enter? '\r' : default;
+                if (((Keys) wParam & Keys.KeyCode) == Keys.Enter)
+                {
+                    // Why \r and not \n? Because it really doesn't matter...
+                    Character = '\r';
```

Comment:
actually it does matter, we need to translate it back ... consider extracting into a constant?